tests/scripts/embedding_log_test.py

```diff
@@ -7,34 +7,34 @@
 from ansys.mechanical.core.embedding.logger import Configuration, Logger
 
 
-def log_before_initialize():
+def log_before_initialize(version):
     """Write a log without initializing the embedded instance."""
     Logger.error("message")
```

Comment:
Why are you passing the version here if it is not used? Do you plan on writing a log message with it?

Reply:
@RobPasMue look at line 55, I treat all the functions in this module the same way, passing the version in.  I'd rather do this than complicate the CLI implementation.

Reply:
However, you are not using it for anything, I'm I wrong? Passing in the ``version`` argument is not neccessary here, IMO

Reply:
Sure, sounds fine!